week-2-practise/level-2/6-map.js

```diff
@@ -2,9 +2,13 @@
 // 1. Write a function that formats the numbers into a string with the percentage symbol appended e.g. "10%"
 // 2. Declare a new array, `percentagesFormatted`, containing
 //    - each item in `percentages` formatted by your function
+function format(item) {
+  return item + "%";
+}
 
 var percentages = [1, 23, 92, 18];
 
+var percentagesFormatted = percentages.map(format);
```

Comment:
Hooray. A happy ending.

Reply:
I nearly lost my patience to this one. :-) thank you Peter.